src/main/msp/msp_protocol.h

```diff
@@ -284,6 +284,7 @@
 #define MSP_VTXTABLE_BAND        137    //out message         vtxTable band/channel data
 #define MSP_VTXTABLE_POWERLEVEL  138    //out message         vtxTable powerLevel data
 #define MSP_MOTOR_TELEMETRY      139    //out message         Per-motor telemetry data (RPM, packet stats, ESC temp, etc.)
+#define MSP_MOTOR_OUTPUT_REORDERING          140    //out message         Array to remap motors
```

Comment:
These should go into `src/main/msp/msp_protocol_v2_betaflight.h`. MSPv1 is well and truly full.

Reply:
@mikeller, oh I did not know v2 was a thing. Will move, thanks!

Reply:
@mikeller , done!